CONTRIBUTING.md

```diff
@@ -76,6 +76,10 @@ Maintainers are:
 ### PR Checklist
 
 Before your PR can be merged it must meet the following criteria:
+* PR Desciption should be informative, explaining what is in it and a high level design in case of new feature
```

Comment:
Nice! I'm glad we are addressing this issue and I think it's a great idea to update this document. That said, I think we should differentiate between merging criteria (need for documentation, unit tests, etc.) and PR quality guidelines. 

I would add these in a separate section and avoid mixing them with the strict merge criteria. Maybe we could also add examples for each one of the fields (for example, I think https://github.com/kubevirt/containerized-data-importer/pull/2709 would fit nicely as an example for this first point).

Reply:
I didn't want to overload the lines but an example can be good.
Regarding putting it in a different section Im not sure cause I do think this should be a criteria before merging a PR not just a quality thing.